pom.xml

```diff
@@ -18,6 +18,7 @@
      This project contains some code from https://github.com/eclipse-jdt/eclipse.jdt.core with some minor changes.
      It does not match our code quality standard and can not be analyzed like other code in this repository.
     -->
+    <sonar.inclusions>pom.xml,src/main/java/empty.xml</sonar.inclusions>
```

Comment:
you can drop this line

Reply:
To have a green quality gate we need to analyze at least one file: `pom.xml`. But `pom.xml` is not anymore analyzed because of this sonar-maven-plugin bug [MSONAR-218](https://sonarsource.atlassian.net/browse/MSONAR-218). So we need to keep temporarily the following hack until  the bug is fixed:
```xml
    <sonar.sources>.</sonar.sources>
    <sonar.inclusions>pom.xml</sonar.inclusions>
```